### ISSUE TITLE
Configurable SSL

### ISSUE THREAD

**Issue by t0ta11ed** [countercept/snake#19]
Allow for configurable SSL for the web interface. The current default HTTP is not very secure when combined with the lack of authentication. If this is already possible, expand on this in documentation.

**Comment by alexkornitzer** [comment]
Understood, this can already be done using something like nginx. Are you using Docker or Bare Metal?

**Comment by t0ta11ed** [comment]
Yes, we are probably going to use Nginx for this but it seems like this could be easily configurable within Snake? We are probably going to bolt on auth too at some point, but will look to contribute if so.

**Comment by alexkornitzer** [comment]
Okay so I had a look into it and while its doable, it just looks to be far easier to do with an nginx proxy. Auth should not be too hard to bolt on depending on the approach, I have kind of back burnered it into v2.0, if I ever get time to start on it.
